modules/ROOT/pages/apikit-error-handling-reference.adoc

```diff
@@ -1,38 +1,64 @@
+| APIKIT:NOT_ACCEPTABLE | Not acceptable
+| APIKIT:UNSUPPORTED_MEDIA_TYPE | Unsupported media type
+| APIKIT:NOT_IMPLEMENTED | Not implemented
+|===
+
+== Error Handling
+
+APIkit scaffolder generates error-handling code based on the most widely-used HTTP status code responses, which are mapped to the error types and messages described as follows:
+
+[%header]
+|===
+| Status Code | Error Type| Response Message
 | 400 | APIKIT:BAD_REQUEST | Bad request
 | 404 | APIKIT:NOT_FOUND | Resource not found
 | 405 | APIKIT:METHOD_NOT_ALLOWED | Method not allowed
 | 406 | APIKIT:NOT_ACCEPTABLE | Not acceptable
 | 415 | APIKIT:UNSUPPORTED_MEDIA_TYPE | Unsupported media type
+| 501 | APIKIT:NOT_IMPLEMENTED | Not implemented
 |===
 
-The following example of the HTTP 400 response shows the construction of the error handler. To add your own custom status code, following the pattern shown here.
+== Example Error
+
+The following code shows an error handler for an HTTP 400 response. Take it as reference to build your own custom status codes and messages.
 
 [source,xml,linenums]
 ----
- <on-error-propagate type="APIKIT:BAD_REQUEST" doc:name="On Error Propagate">
-                <ee:transform xmlns:ee="http://www.mulesoft.org/schema/mule/ee/core" xsi:schemaLocation="http://www.mulesoft.org/schema/mule/ee/core http://www.mulesoft.org/schema/mule/ee/core/current/mule-ee.xsd">
-                    <ee:message>
-                        <ee:set-payload><![CDATA[%dw 2.0
-output application/json
----
-{message: "Bad request"}]]></ee:set-payload>
-                    </ee:message>
-                    <ee:variables>
-                        <ee:set-variable variableName="httpStatus">400</ee:set-variable>
-                    </ee:variables>
-                </ee:transform>
-            </on-error-propagate>
+...
+    <on-error-propagate type="APIKIT:BAD_REQUEST" doc:name="On Error Propagate">
+        <ee:transform xmlns:ee="http://www.mulesoft.org/schema/mule/ee/core" xsi:schemaLocation="http://www.mulesoft.org/schema/mule/ee/core http://www.mulesoft.org/schema/mule/ee/core/current/mule-ee.xsd">
+            <ee:message>
+                <ee:set-payload><![CDATA[%dw 2.0
+    output application/json
+    ---
+    {message: "Bad request"}]]></ee:set-payload>
+            </ee:message>
+            <ee:variables>
+                <ee:set-variable variableName="httpStatus">400</ee:set-variable>
+            </ee:variables>
+        </ee:transform>
+    </on-error-propagate>
+...
 ----
 
-Every time a message throws an error, APIkit checks to see if the error matches the value of any on-error-propagate types defined in the project. The main flow in an APIkit project sends an HTTP error response consisting of an HTTP status code and corresponding plain-language message.
\ No newline at end of file
+The main flow transmits an HTTP error response that encompasses an HTTP status code paired with an associated message expressed in clear and straightforward language.
```

Comment:
Which message is clear and straightforward? I don't see the referenced message.

Reply:
Requesting assistance from SMEs. Thanks!

Reply:
The clear and straightforward message is the one included in the body of the message. In this case, it's the “message: “Bad request”" that appears in the body.